.tekton/release-service-pull-request.yaml

```diff
@@ -273,13 +280,11 @@ spec:
         value: $(tasks.clone-repository.results.commit)
       - name: IMAGE_EXPIRES_AFTER
         value: $(params.image-expires-after)
-      - name: ALWAYS_BUILD_INDEX
```

Comment:
The param is removed from the pipeline now, which is what I was saying, thanks for that. Your original version of this PR had this (and the following line) changed to
```
      - name: ALWAYS_BUILD_INDEX
        value: 'true'
```
I think you updated it based on my unclear comment. So you probably want to readd this line and the value true part?

Reply:
Maybe you didn't `git add` it after? It is still showing to me as the `ALWAYS_BUILD_INDEX` task param is removed